tests/common_testing/alfasim_sdk_common_testing/filled_case_descriptions.py

```diff
@@ -441,8 +441,10 @@
 )
 CASE_OUTPUT_DESCRIPTION = case_description.CaseOutputDescription(
     trends=TRENDS_OUTPUT_DESCRIPTION,
+    automatic_trend_frequency=True,
```

Comment:
What is the idea of these parameters? If they are `True`, the `trend_frequency` and `profile_frequency` are ignored?

Reply:
They already exist, but are missing in some places. If I understand them correctly, it's like what you said, `automatic_...` being `True` means the property will be derived automatically from some defaults.

Reply:
It is also possible that the "case description"-"alfacase" round trip is saving the unused values. I remember to add that option with the multi inputs to test transient and constant in one go (but disabled on production).

But as he pointed out "... used is the highest value between: default profile frequency, maximum timestep configured, and an estimate ..."